releases/unreleased/support-projects.yml

```diff
@@ -0,0 +1,19 @@
+---
+title: Support Projects
+category: added
+author: Miguel Ángel Fernández <mafesan@bitergia.com>
+issue: 42
+notes: >
+    This feature allows to add Projects to Bestiary.
+
+    Projects are meant to represent an abstract set of end-point
+    views sharing a common context. The projects can also have
+    n sub-projects which also have sets of end-point views.
```

Comment:
This is not clear at all. Can we have a better definition?

Reply:
Ok, I'm working on it!

Reply:
New definition:

> Projects are defined as a set of data locations which have to be grouped under the same entity. Moreover, this grouping may have a hierarchy by defining sub-projects.

WDYT?

Reply:
Good!